official/resnet/README.md

```diff
@@ -86,19 +86,39 @@ num_gpus:
 + 2+: Use MirroredStrategy (data parallelism) to distribute a batch between devices.
 
 ### Pre-trained model
-You can download 190 MB pre-trained versions of ResNet-50. Reported accuracies are top-1 single-crop accuracy for the ImageNet validation set. Simply download and uncompress the file, and point the model to the extracted directory using the `--model_dir` flag.
-
-ResNet-50 v2 (Accuracy 76.05%):
-* [Checkpoint](http://download.tensorflow.org/models/official/20180601_resnet_v2_imagenet_checkpoint.tar.gz)
-* [SavedModel](http://download.tensorflow.org/models/official/20180601_resnet_v2_imagenet_savedmodel.tar.gz)
-
-ResNet-50 v2 (fp16, Accuracy 75.56%):
-* [Checkpoint](http://download.tensorflow.org/models/official/20180601_resnet_v2_fp16_imagenet_checkpoint.tar.gz)
-* [SavedModel](http://download.tensorflow.org/models/official/20180601_resnet_v2_fp16_imagenet_savedmodel.tar.gz)
-
-ResNet-50 v1 (Accuracy 75.91%):
-* [Checkpoint](http://download.tensorflow.org/models/official/20180601_resnet_v1_imagenet_checkpoint.tar.gz)
-* [SavedModel](http://download.tensorflow.org/models/official/20180601_resnet_v1_imagenet_savedmodel.tar.gz)
+You can download pre-trained versions of ResNet-50. Reported accuracies are top-1 single-crop accuracy for the ImageNet validation set.
+Models are reported as both checkpoints produced by Estimator during training, and as SavedModels which are more portable. Both ResNet v1
+and ResNet v2 have been trained in both fp16 and fp32 precision. (Here v1 refers to "v1.5". See the note above.) Furthermore, SavedModels
+are generated to accept either tensor or JPG inputs, and with channels_first (NCHW) and channels_last (NHWC) convolutions to better
+accommodate both CPU and GPU architectures.
+
+ResNet-50 v2 (fp32, Accuracy 76.47%):
+* [Checkpoint](http://download.tensorflow.org/models/official/20181001_resnet/checkpoints/resnet_imagenet_v2_fp32_20181001.tar.gz)
+* SavedModel [(NCHW)](http://download.tensorflow.org/models/official/20181001_resnet/savedmodels/resnet_v2_fp32_savedmodel_NCHW.tar.gz),
```

Comment:
(feel free to ignore this suggestion)

it may be more readable to write NCHW -> NCHW/GPU and NHWC -> NHWC/CPU,
so folks wanting to download the saved model, can quickly do so without attempting
to read the text above and map one of these formats to GPU vs CPU

Reply:
I think I'm going to leave it as is simply because MKL wants NCHW which blurs the line.

Reply:
fair enough.